docs/reference/setup-tools/kubeadm/kubeadm-alpha.md

```diff
@@ -190,14 +190,14 @@ or selectively configure single elements.
 {% include_relative generated/kubeadm_alpha_phase_bootstrap-token_cluster-info.md %}
 {% endcapture %}
 
-{% capture bootstrap-token_node_allow-auto-approve %}
-{% include_relative generated/kubeadm_alpha_phase_bootstrap-token_node_allow-auto-approve.md %}
-{% endcapture %}
-
 {% capture bootstrap-token_node_allow-post-csrs %}
 {% include_relative generated/kubeadm_alpha_phase_bootstrap-token_node_allow-post-csrs.md %}
 {% endcapture %}
 
+{% capture bootstrap-token_node_allow-auto-approve %}
+{% include_relative generated/kubeadm_alpha_phase_bootstrap-token_node_allow-auto-approve.md %}
+{% endcapture %}
+
```

Comment:
I don't think this is a valid change. It looks to me that auto approving has to be enabled before allowing to send CSRs. Need some clarification why the sequence need a change.

Reply:
I made this adjustment based on the description of this link. https://kubernetes.io/docs/reference/setup-tools/kubeadm/implementation-details/#configure-tls-bootstrapping-for-node-joining

The sequence is like this: 
1. Create a bootstrap token.
2. Allow joining nodes to call CSR API
3. Setup auto approval for new bootstrap tokens
...